cdap-e2e-tests/src/e2e-test/features/namespaceadmin/NSAdminDesignTime.feature

```diff
@@ -0,0 +1,123 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+@Namespaceadmin
+Feature: NameSpaceAdmin - Validate nameSpace admin design time scenarios
+
+  @Namespaceadmin
+  Scenario:Verify user is able to click on the namespace admin tab and successfully navigates to the page
+    Given Open Datafusion Project to configure pipeline
+    When Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Verify if user successfully navigated to namespace admin page
+
+  @Namespaceadmin
+  Scenario:Validate user is able to open compute profile page and create a profile for selected a provisioner
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Click on create profile button for "default" Namespace
+    Then Select a provisioner: "remoteHadoopProvisioner" for the compute profile
+    Then Verify the Create a Profile page is loaded for selected provisioner
+    Then Enter input plugin property: "profileLabel" with value: "validProfile"
+    Then Enter textarea plugin property: "profileDescription" with value: "validDescription"
+    Then Enter input plugin property: "host" with value: "testHost"
+    Then Enter input plugin property: "user" with value: "testUser"
+    Then Enter textarea plugin property: "sshKey" with value: "testSSHKey"
+    Then Click on: "Create" button in the properties
```

Comment:
This seems to be a case of locating the button using the text content of the button. We should avoid this.